wire-runtime-swift/src/test/swift/RedactableTests.swift

```diff
@@ -72,4 +72,24 @@ final class RedactableTests: XCTestCase {
             RedactedLargeMessage.RedactedKeys.a
         )
     }
+    
+    func testLargeMessageRedactedUnsafeNameField() {
+        let redacted = RedactedLargeMessage {
+            $0.description_ = "foo"
```

Comment:
Please add `Codable` tests around `description_`, which should use the JSON key of `"description"`

Reply:
done